core/src/main/java/it/pagopa/selfcare/party/registry_proxy/core/AOOService.java

```diff
@@ -3,11 +3,12 @@
 import it.pagopa.selfcare.party.registry_proxy.connector.model.AOO;
 import it.pagopa.selfcare.party.registry_proxy.connector.model.QueryResult;
 
+import java.util.List;
+
 
 public interface AOOService {
 
     QueryResult<AOO> findAll(int page, int limit);
-
-    AOO findByUnicode(String codiceUniAOO);
+    AOO findByUnicode(String codiceUniAOO, List<String> categoriesList);
```

Comment:
should be named `findByUnicodeAndCategories`

Reply:
il filtro per categoria non è obbligatorio. cambiamo comunque il nome?

Reply:
that's ok, let's leave it like this